### PR TITLE
Fixed

### PR DESCRIPTION
**Pull request by awsirkis**

### CODE REVIEW

File: assignment3/excercise-one.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+let string_arr = ["apple", "bannana", "coconut", "grape", "orange"];
```
Comment (jimmynono) [inline assignment3/excercise-one.js]: Tip:

Layout on JS files is a bit hard to describe, but what's good to do is put a space after a group of variable declarations, and after a group of code...What I would do is put a space after all of these lines, or mabye only after the console.logs. It just makes it a bit easier to read. It's a fine art, and after a while you'll get the feel of where white space is to go.

```
let x = 32;
let y = 112;

console.log(x + y);

let z = 'demo';

console.log(z);
```

Something like that

File: assignment3/excercise-three.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+let n = ['a', 'b', 'c', 'd', 'e'];
+for(i = 0; i < n.length; i++) {
```
Comment (jimmynono) [inline assignment3/excercise-three.js]: you'll want a blank line above the for loop and a space after the for keyword

File: assignment3/excercise-three.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  console.log(String(i) + ',' + String(n[i]));
```
Comment (jimmynono) [inline assignment3/excercise-three.js]: these String() methods aren't really needed, but not a terrible idea.

File: lesson4/assignment1/index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<html>
+<body>
+
+<h2>Use JavaScript to Change Text</h2>
+<p>This example writes "Hello JavaScript!" into an HTML element with id="demo":</p>
+
+
+<p id="demo" onclick="page_mult()">Click Me to Multiply Numbers!</p>
+
+<script>
```
Comment (jimmynono) [inline lesson4/assignment1/index.html]: Please put all JS in a javascript file. No inline JS.

Comment (jimmynono) [reply lesson4/assignment1/index.html]: Actually...that's OK. the assignment says you can...but that was an oversight on my part. In the future put all of your JS in js files. =) Sorry about that

File: lesson4/assignment1/script.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (jimmynono) [inline lesson4/assignment1/script.js]: Make sure that js files are indeed JS files.

File: assignment3/excercise-two.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+let n = prompt("Sentence:");
+let sentence = n.split(' ');
+while(!(sentence.length >= 5 && sentence[sentence.length].slice(-1) == '.')){
```
Comment (ClovisI) [inline assignment3/excercise-two.js]: This sentence[sentence.length].slice(-1) is giving me undefined errors.  I think you need to debug this section, particularly, the sentence[sentence.length] section.  You need to actually subtract one here so it's:  sentence[sentence.length-1]

Then your logic will work.  I like this idea though.  Great work.

**Pull request closed without merging**